### PR TITLE
fix: 테스트 케이스 수정

### PR DESCRIPTION
**Pull request by newkayak12**
## 🎫 Context

- github:
- jira:

## 📄 Summary
- edge case 수정

## ✨ What’s Changed
- docker 및 flyway 수정
- 테스트케이스 수정

## 🧩 Motivation & Background
- ~해서 필요

## 🔥 Impact
- ~한 변경점이 생김

## 🚦 How to Test (검증방법)
- ~를 테스트

## 📝 Notes

- 참고/리뷰 요청/논의 사항 등

### CODE REVIEW

File: batch-module/src/test/kotlin/com/reservation/ReservationBatchApplicationTest.kt [hunk 1]
```diff
@@ -36,6 +38,9 @@ class ReservationBatchApplicationTest {
             registry.add("spring.datasource.url") { mysqlContainer.jdbcUrl }
             registry.add("spring.datasource.username") { mysqlContainer.username }
             registry.add("spring.datasource.password") { mysqlContainer.password }
+
+            registry.add("spring.data.redis.host") { redisContainer.host }
+            registry.add("spring.data.redis.port") { redisContainer.getMappedPort(6379) }
         }
     }
```
[the remark this answers is not in the document]
Comment (newkayak12) [reply batch-module/src/test/kotlin/com/reservation/ReservationBatchApplicationTest.kt:46]: spring data redis host가 맞음
